src/stub_launcher.cc

```diff
@@ -72,12 +76,13 @@ StubLauncher::Initialize(ModelState* model_state)
 
   model_version_ = model_state->Version();
 
+  // Should check for model.py only when not using platform models.
```

Comment:
can remove comment - maybe move to commit message. If I understand correctly the check is now moved to stub